labs/chaincode-analyzer.md

```diff
@@ -0,0 +1,29 @@
+# Lab Name
+Chaincode Analyzer
+
+# Short Description
+A static tool to check and detect potentially risky codes in smart contracts (a.k.a, chaincodes).
+
+
+# Scope of Lab
+The aim of Chaincode Analyzer is to provide the way to check whether the smart contracts includes potentially risky codes or not.
+The tool performs static analysis on chaincode source codes written in Golang and detects potentially risky parts such as generating random values, accessing outside of blockchain networks and iteration on map object.
+For more details of the tool and target risks:
+https://github.com/FujitsuLaboratories/ChaincodeAnalyzer
+
+In Hyperledger Fabric, developers can use general purpose programming languages, such as JavaScript, Golang and Java, to implement their chaincodes.
+Compared to using domain specific languages (DSLs) like Solidity of Ethereum, the efforts of implementing smart contracts are smaller since develpers do not need to study new language for smart contracts.
```

Comment:
Typo develpers->developers

Reply:
Thank you for your review. I fixed the typos at e451d55.